source/utils/sentenceHelper.py

```diff
@@ -11,16 +11,24 @@
 FULL_WIDTH_TERMINATORS_PATTERN: re.Pattern = re.compile("[。！？]")
 
 
-def findEndOfSentence(s: str, offset: int) -> int:
+def findEndOfSentence(sentence: str, offset: int) -> int:
+	"""
+	Returns an index which points one character past the end of the current sentence,
+	or if the sentence is at the end of the string, returns the length of the string.
+	In the case of western sentence terminators,
+	conbinations of space, \r, or \n at the end of the sentence are included.
+	@param sentence: string containing at least current sentence
+	@param offset: int indicating where to start searching for the end of the sentence
+	"""
 	res = -1
```

Comment:
I think it would be safer if this returned None if no end is found

```suggestion
	res = None
```

Reply:
Returning None is safer? How? str.find returns -1 if search string is not found. Isn't this similar to a string find function?

Reply:
Rather than relying on a special value, it would be clearer if it was a different type, and a type which more accurate represents the return result. -1 refers to the end of a list in python, which means the return result may be accidentally used incorrectly. If no sentence end is found, then we have no index, i.e. `None`.